content/management/configuration-api/index.mdx

```diff
@@ -2413,7 +2414,7 @@ Here's what you need to know about **webhooks**:
 
 ### Register Webhook
 
-Registers a webhook for the Client ID (application) provided in the request. Later on, registered webhooks can be [enabled](#enable-license-webhooks) for a given license.
+Registers a webhook for the Client ID (application) provided in the request. Later on, registered webhooks can be [enabled](#enable-license-webhooks) for a given license. Number of webhooks that you can register is limited to three for every action.
```

Comment:
Shouldn't we introduce this description in `3.3` and higher?

Reply:
I have added it in 3.4 [link]( https://developers.labs.livechat.com/docs/feature/API-5631-limit-webhook-registration/management/configuration-api/v3.4/#webhooks)
and 3.3 [link](https://developers.labs.livechat.com/docs/feature/API-5631-limit-webhook-registration/management/configuration-api/#webhooks)
do you mean some other place?

Reply:
And also here I forgot that we switched to 3.3 as default